core/frontend/src/IModelApp.ts

```diff
@@ -121,6 +121,10 @@ export interface IModelAppOptions {
   rpcInterfaces?: RpcInterfaceDefinition[];
   /** @beta */
   realityDataAccess?: RealityDataAccess;
+  /** If present, supplies the publicURL.
+   * URLs targetting content under the /public/ folder should prepend this with a trailing `/`.
```

Comment:
targeting instead of targetting

The way this is worded is kind of strange, do you mean to say they need a `/` at both the start and end of the path?

Reply:
updated